docs/using_the_api.md

```diff
@@ -171,6 +212,16 @@ query {
 }
 ```
 
+The comparator flag allows you to specify how you want your filter applied. Specifically, we let you use the use:
+
+* greater than (gt)
+* equal to (eq)
+* less than (lt)
+
+All comparators need to be passed in as strings. For example, to filter by the year `1970`, you would pass it as:
```

Comment:
All filter values need to be passed in as strings -- clear up what needs to be a string